src/main/java/create_task_use_case/CreateTaskInteractor.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Create Task Interactor Class.
+ * Author: Fardin Faruk
+ * Modified By: Fardin Faruk
+ * Created: Nov 19, 2022
+ * Last Modified: Nov 26, 2022
+ */
+package create_task_use_case;
+
+import entities.Task;
+import entities.TaskFactory;
+
+public class CreateTaskInteractor implements CreateTaskInputBoundary {
+    final CreateTaskDsGateway createTaskDsGateway;
+    final CreateTaskOutputBoundary createTaskOutputBoundary;
+    final TaskFactory taskFactory;
+
+    public CreateTaskInteractor(TaskFactory taskFactory, CreateTaskOutputBoundary createTaskOutputBoundary,
+                                CreateTaskDsGateway createTaskDsGateway){
+        this.taskFactory = taskFactory;
+        this.createTaskOutputBoundary = createTaskOutputBoundary;
+        this.createTaskDsGateway = createTaskDsGateway;
+    }
+
+
+    @Override
+    public CreateTaskOutputData create(CreateTaskInputData createTaskInputData) {
+        if (createTaskDsGateway.existsByTitle(createTaskInputData.getTitle(), createTaskInputData.getDayID())){
+            return createTaskOutputBoundary.prepareFailView("Task with the same name already exists.");
```

Comment:
Error message would be more accurate if it noted task already exists _in the current day_.

Reply:
Raghav's comment phrases the necessary changes much better. Resolving this one for (hopefully) easier navigation.